config/src/aws-net-std__ipv4.adb

```diff
@@ -0,0 +1,819 @@
+------------------------------------------------------------------------------
+--                              Ada Web Server                              --
+--                                                                          --
+--                     Copyright (C) 2000-2018, AdaCore                     --
```

Comment:
2018 only?

Reply:
I remain the copyright because it is bit modified old config/src/aws-net-std__gnat.adb. If you still think that copyright have to be just 2018 I'll change, it's does not really matter for me.

Reply:
No, makes sense.